vindbaarheid/vindbaarheid.shacl.ttl

```diff
@@ -0,0 +1,492 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix premis: <http://www.loc.gov/premis/rdf/v3/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix meemoo: <http://data.meemoo.be/ns/>  .
+@prefix ebu: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+##############################
+#         NodeShapes         #
+##############################
+
+####################
+#      PREMIS      #
+####################
+premis:IntellectualEntity a rdfs:Class, sh:NodeShape ;
+    #? max en minCount constrains?
+    sh:property meemoo:identifierShape, meemoo:nameShape, meemoo:descriptionShape,
+    [
+        a sh:PropertyShape ;
+        sh:path premis:identifier ;
+        sh:class meemoo:LocalIdentifier ;
+        #? Aantal mogelijk?
+        sh:maxCount 2 ; #? Warning hierover? Meerdere PropertyShapes uiteen halen voor speciefiekere errors?
```

Comment:
minimaal 0 en maximaal ongelimiteerd voor een violation.
Je kan eventueel een extra shape definieren die een warning geeft als er niet minstens 1 is. Maar die 2 lijkt mij arbitrair.